docs/examples/openiotsdk_examples.md

```diff
@@ -16,36 +16,54 @@ submodules using the following command:
 $ git submodule update --init
 ```
 
-The VSCode devcontainer has all dependencies pre-installed. Using the VSCode
-devcontainer is the recommended way to interact with Open IoT SDK port of the
-Matter Project. Please read this
+The VSCode devcontainer has all the dependencies pre-installed. Using the VSCode
+devcontainer is the recommended way to interact with the Open IoT SDK port of
+the Matter Project. Please read this
 [README.md](../../..//docs/VSCODE_DEVELOPMENT.md) for more information.
 
+There are also some python packages that are required which are not provided as
+part of the VSCode devcontainer. To install these run the following command from
+the CLI:
+
+```
+${MATTER_ROOT}/scripts/run_in_build_env.sh './scripts/build_python.sh --install_wheel
+build-env'
+```
```

Comment:
I think this can be a separate chapter e.g `Testing setup` - it's only for testing purposes. And there is a build and install the Python Matter controller step which is required for the integration tests suite.  We can link the details: https://github.com/ARM-software/connectedhomeip/blob/development/docs/guides/python_chip_controller_building.md